packages/graphback-core/src/plugin/GraphbackCoreMetadata.ts

```diff
@@ -45,11 +52,17 @@ export class GraphbackCoreMetadata {
 
         //Contains map of the models with their underlying CRUD configuration
         this.models = [];
+        const relationships = [];
         //Get actual user types 
         const modelTypes = this.getGraphQLTypesWithModel();
         for (const modelType of modelTypes) {
-            const model = this.buildModel(modelType)
-            this.models.push(model)
+            const relationshipMetadata = this.buildRelationships(modelType);
+            relationships.push(...relationshipMetadata);
```

Comment:
```suggestion
            relationships.push(relationshipMetadata);
```

Reply:
This wouldn't work. Here we are merging two arrays with spread operator. Without it you would get an array inside an array.